src/SDKs/Kusto/Management.Kusto/Microsoft.Azure.Management.Kusto.csproj

```diff
@@ -5,7 +5,7 @@
 
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Kusto</PackageId>
-    <Description>Provides developers with libraries for the updated kusto platform under Azure Resource manager to create ane manage clusters. Note: This client library is for Virtual Machines under Azure Resource Manager.</Description>
```

Comment:
Description cannot be changed without bumping the version number here. What's the intent of this change?

Reply:
The current description is wrong.